lib/features/support/resilient_driver.rb

```diff
@@ -0,0 +1,63 @@
+require 'appium_lib'
+require_relative './fast_selenium'
+require_relative './logger'
+
+# Handles Appium driver restarts and retries in the case of failure
+class ResilientAppiumDriver
+  # Creates the ResilientAppiumDriver
+  #
+  # @param username [String] the BrowserStack username
+  # @param access_key [String] the BrowserStack access key
+  # @param local_id [String] the identifier for the BrowserStackLocal tunnel
+  # @param target_device [String] a key from the Devices array selecting which device capabilities to target
+  # @param app_location [String] the location of the test-app to upload
+  # @param locator [Symbol] the primary locator strategy Appium should use to find elements
+  # @param additional_capabilities [Hash] a hash of additional capabilities to be used in this test run
+  def initialize(username, access_key, local_id, target_device, app_location, locator = :id, additional_capabilities = {})
+    @driver = AppAutomateDriver.new username,
+                                    access_key,
+                                    local_id,
+                                    target_device,
+                                    app_location,
+                                    locator,
+                                    additional_capabilities
+  end
+
+  def respond_to_missing?(method_name, include_private = false)
+    @driver.respond_to_missing? method_name
+  end
+
+  # rubocop: disable MethodMissingSuper
+  def method_missing(method, *args, &block)
+    raise "Method '#{method}' does not exist" unless @driver.respond_to?(method)
```

Comment:
I think something like this will let you get rid of the rubocop suppression; it looks a bit weird though because the `super` will immediately fail

```suggestion
    return super unless @driver.respond_to?(method)
```

Reply:
Yep, I like that - I'm comfortable with the wrapping class behaving just like the inner one when called incorrectly.